src/pages/workspace/WorkspaceResetBankAccountModal.js

```diff
@@ -50,26 +37,14 @@ const WorkspaceResetBankAccountModal = (props) => {
             ) : props.translate('workspace.bankAccount.clearProgress')}
             danger
             onCancel={BankAccounts.cancelResetFreePlanBankAccount}
-            onConfirm={() => BankAccounts.resetFreePlanBankAccount()}
+            onConfirm={() => BankAccounts.resetFreePlanBankAccount(bankAccountID)}
             shouldShowCancelButton
-            isVisible={lodashGet(props.reimbursementAccount, 'shouldShowResetModal', false)}
```

Comment:
Why are we removing this condition for `isVisible`?

Reply:
Because now, we condition it from the parent view and early return when showing it.

<img width="951" alt="image" src="https://user-images.githubusercontent.com/2463975/211591165-6504febb-58dc-40c9-97d2-795f4ac18077.png">